torch/fx/experimental/proxy_tensor.py

```diff
@@ -283,6 +288,10 @@ def set_proxy_slot(  # type: ignore[no-redef]
         # We DO want to clobber proxies, with a similar rationale as for tensors.
         assert isinstance(proxy, Proxy)
         tracer.script_object_tracker[obj] = proxy
+    elif is_opaque_value(obj):
```

Comment:
Sorry, do you happen to have a test case that runs into the case where these changes are needed?

Reply:
Sorry ignore the comment ab updating the test -- instead of creating a new tracker for opaque objects can we combine it with the ScriptObject type code? When we trace symbolically we convert the opaque objects to FakeScriptObjects, so when we trace with real I think we can also just use the same script object trackers. I think the change you need to make is just replace all `isinstance(obj, _AnyScriptObject)` with `isinstance(obj, _AnyScriptObject) or is_opaque_value(obj)`